### ISSUE TITLE
Tab characters in body strings break code snippets

### ISSUE THREAD

**Issue by kkwillmert** [Microsoft/vscode-mssql#874]
User code snippets I had set up on a previous version of the mssql extension that contained tab characters in the body strings no longer work, and when viewing the sql.json file for the snippets, I get  a red squiggly error indicator on the first body string that contains a tab character.  Removing the tab from that string will cause the red squiggly to show up on the next body string with a tab.

- MSSQL Extension Version: 1.0.0
- VSCode Version: 1.12.1
- OS Version: Windows 7 Enterprise

Steps to Reproduce:

1. Open the sql.json file for SQL user snippets.

2. Add the following, using tab characters for the indentation within the strings:
```
"Query information schema": {
	"prefix": "INFO",
	"body": [
		"SELECT",
		"	*",
		"FROM",
		"	INFORMATION_SCHEMA.COLUMNS",
		"",	
		"	INNER JOIN INFORMATION_SCHEMA.TABLES",
		"	ON INFORMATION_SCHEMA.COLUMNS.TABLE_NAME = INFORMATION_SCHEMA.TABLES.TABLE_NAME",
		"	AND INFORMATION_SCHEMA.TABLES.TABLE_TYPE <> 'VIEW'",
		"WHERE",
		"	INFORMATION_SCHEMA.COLUMNS.COLUMN_NAME = '$0'",
		"	--INFORMATION_SCHEMA.COLUMNS.TABLE_NAME = ''",
		";"
	],
	"description": "Query on information schema for table/column names."
}
```

3. Save changes and attempt to use the snippet.

**Comment by kevcunnane** [comment]
@kkwillmert I believe this is an issue in the VSCode snippet support. Taking a quick search over their issues, it appears that you should use `\t` instead of the tab character to have this handled correctly. This will also ensure users tabs vs. spaces preferences are respected. I'd like to close this as external if you have no objections?

**Comment by kkwillmert** [comment]
No objections at all, and thanks for researching and passing along this
info!  I had assumed it was an mssql issue since I encountered it
immediately after I updated mssql, but a few days after I last updated
VSCode.  Probably I just hadn't used that particular snippet since
installing the VSCode update.

For the record, I love the mssql extension, it has entirely replaced my use
of SSMS for my day-to-day quick queries :-)

Thanks again!
-Kris
